### ISSUE TITLE
Error after uploading avatar

### ISSUE THREAD

**Issue by bd0bd** [backdrop-contrib/privatemsg#4]
Fresh installed again.

I have added a photo at /user/1/edit and then got this error at /user/1 (1 — this is an admin, me)

```
Warning: Parameter 1 to privatemsg_user_update() expected to be a reference, value given in     module_invoke_all() (line 945 of /home/www/domain.com/core/includes/module.inc).
```

**Comment by Graham-72** [comment]
I think this is because of the change to hook_user_update in Backdrop and so I am trying to fix it with commit 82e472b to privatemsg.module.

**Comment by bd0bd** [comment]
Yes, it is fixed.
Thank you!

By the way, I use only Private messages module at this time. I do not use any other (Privatemsg filter etc).
